wocg/create_project.py

```diff
@@ -77,13 +80,20 @@ def get_new_project(project_name, repository):
 
 def get_new_component(
         project, repository, branch, addon_name, tmpl_component_slug,
-        addons_subdirectory=None):
+        addons_subdirectory=None, git_export_base_url=None):
     po_file_mask = '{}/i18n/*.po'.format(addon_name)
     pot_filepath = '{addon_name}/i18n/{addon_name}.pot'.format(
         addon_name=addon_name)
+    git_export = ''
+    component_slug = get_component_slug(project, addon_name)
     if addons_subdirectory:
         po_file_mask = os.path.join(addons_subdirectory, po_file_mask)
         pot_filepath = os.path.join(addons_subdirectory, pot_filepath)
+    if git_export_base_url:
+        git_export = urljoin(
+            git_export_base_url,
+            os.path.join('git', project.slug, component_slug)
```

Comment:
My ideas was to use the project slug only here (the name is even nicer, actually), so the git url refers to the project and not a specific component, since the git ref is shared across all components.

Reply:
so base_url + git + project_name only ?

Reply:
yep